package-dev/Tests/Runtime/TestSentryOptions.json

```diff
@@ -0,0 +1 @@
+{"enabled":true,"captureInEditor":true,"dsn":"http://test.com","debug":true,"debugOnlyInEditor":true,"diagnosticsLevel":3,"requestBodyCompressionLevel":1,"attachStacktrace":true,"sampleRate":1,"release":"release","environment":"test"}

```

Comment:
should be this be in gitignore?

Reply:
If we ship the package pre-configured wouldn't that be a good thing?
Install Package -> Hit Play -> Events get already logged on our Unity-Project?

Reply:
This one seems to be only used on tests though (based on the name). I believe it's created during Play tests.

Reply:
This is used in tests only, yes. Seems ok to be `.gitignore`d.